src/reducers/bastions.js

```diff
@@ -7,7 +7,8 @@ export default function bastions(state = [], action) {
     case 'CREATE_BASTION_SUCCESS':
       return [...state, action.newBastion];
     case 'DELETE_BASTION_SUCCESS':
-      return state.filter(bastion => bastion.name !== action.bastionName);
+      console.log('in bastion reducer:', action, state)
+      return state.filter(bastion => bastion.StackName !== action.bastionName);
```

Comment:
Changed to use correct property name.  Fixed issue with deleting instances not re-rendering